x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -76,6 +76,54 @@ export const alertComment: Comment = {
   version: 'WzQ3LDFc',
 };
 
+export const hostIsolationComment: Comment = {
```

Comment:
same comment on all these mocks - do not define them as singletons. instead, create a function that returns the `Comment`.  Doing this can cause tests to fail and be very hard to determine why